ide/editor.settings.storage/src/org/netbeans/modules/editor/settings/storage/api/EditorSettings.java

```diff
@@ -207,22 +207,22 @@ public abstract void setHighlightings (
     );
     
     /**
-     * Returns annotations properties for given profile or null, if the 
+     * Returns annotations properties for given profile or an empty Map, if the 
      * profile is not known.
      *
      * @param profile a profile name
-     * @return annotations properties for given profile or null
+     * @return annotations properties for given profile or an empty map
      */
     public abstract Map<String, AttributeSet> getAnnotations (
```

Comment:
the impl never returned null in the last ~20 years. Since it is friend API I decided to change the doc instead of the impl since it seemed to be lower risk.

Reply:
Seems reasonable...